ACM_General/home/static/home/scss/404.scss

```diff
@@ -0,0 +1,62 @@
+body, html {
+  margin: 0;
+  height: 100%;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  -moz-font-feature-settings: "liga" on;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.top-content {
+  /*margin: auto;*/
```

Comment:
If you need it for something, put a comment as to why; Otherwise, please remove for slimmer code.